source/schema.txt

```diff
@@ -338,3 +338,103 @@ To delete a location filter from the map:
 .. |query-build-ref| replace:: :ref:`build-query`
 
 .. |query-build-type| replace:: :ref:`build-query`
+
+Export Your Schema
+------------------
+
+You can export your schema after analyzing it. This is useful for 
+sharing your schema and for comparing schemas across collections.
+
+If you have not already done so, analyze your schema:
+
+1. Select your desired collection and click the :guilabel:`Schema` tab.
+
+#. Click :guilabel:`Analyze Schema`.
+
+.. note::
+   |compass-short| analyzes schemas using :ref:`sampling <sampling>`.
+
+Once your schema has been analyzed, export your schema:
+
+1. In the top menu bar, click :guilabel:`Collection`.
+
+#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+
+.. figure:: /images/compass/share-schema-json.png
+   :alt: Image showing Collection dropdown
+
+Your schema is copied to your clipboard as a |json| object with 
+properties ``fields`` and ``count``:
+
+- ``fields`` is an array of metadata for each analyzed field, including 
+  the field's ``name``, ``types``, and within ``types``, an array of 
+  the actual values analyzed for each type.
+
+- ``count`` is the number of documents :ref:`sampled <sampling>` from 
+  the collection.
+
+The following contrived example shows a schema for a collection 
+of 3 documents with only ``_id`` and ``user`` fields:
```

Comment:
Just curious why we're using such a simple example? It might be more interesting to show a schema with embedded objects, and possibly arrays with mixed types.

Also, if the example only has three documents, we might as well provide them to the user in case they want to import / play around with them themselves.

Reply:
Good ideas, making it a bit more complex to show embedded data and adding the documents to import.

The simple example was to keep the schema output simple so that users can easily see how documents map to it. I think with anything like a real collection the schema is pretty massive and harder to understand without really scrutinizing.